src/content/protocols/spark.md

```diff
@@ -0,0 +1,414 @@
+| ProxyAdmin                                                              | transferOwnership                     | Updates the owner of the ProxyAdmin contract: the account with the rights to change the admin of the proxy and upgrade the implementation contracts.                                                                                                                                                                                                                                                                                                                                             | SubProxy ("Spark Proxy")                                                        |
+| ProxyAdmin                                                              | changeProxyAdmin                      | Transfers ownership over the contract to another address. The new address will have upgrade rights over all upgradeable contracts that have this contract as admin immutable, which only concerns the `V3InterestRatesFactory`.                                                                                                                                                                                                                                                                  | SubProxy ("Spark Proxy")                                                        |
+| ProxyAdmin                                                              | upgrade                               | Upgrades a contract that has this ProxyAdmin as admin. To date this can only upgrade the `V3InterestRatesFactory`.                                                                                                                                                                                                                                                                                                                                                                               | SubProxy ("Spark Proxy")                                                        |
+| ProxyAdmin                                                              | upgradeAndCall                        | Upgrades a contract that has this ProxyAdmin as admin and then calls a function in it. To date this can only upgrade the `V3InterestRatesFactory`.                                                                                                                                                                                                                                                                                                                                               | SubProxy ("Spark Proxy")                                                        |
+
+## Dependencies
+
+Spark has a core dependency in the Sky Protocol, a stage 0 protocol. This dependency is clear from the fact that all key permissions in Spark contracts are held by the Sky Governance.
```

Comment:
two levels of Sky dependency:
1. Sky governance
2. USDS as Sparks central stablecoin, if USDS blows up, Spark blows up